Sources/Vehicle/Extensions/ServerManagerExtension.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+import Shared
+
+public extension ServerManager {
+    func isConnected() -> Bool {
```

Comment:
you don't need to check connected status for servers; do the things you intend to do with or without it being connected. the ui should respond to that state updates.